components/gitpod-db/src/typeorm/migration/1658144661198-WorkspaceInstanceUsageAddExtraFields.ts

```diff
@@ -1,18 +1,22 @@
-import {MigrationInterface, QueryRunner} from "typeorm";
-import {columnExists} from "@gitpod/gitpod-db/lib/typeorm/migration/helper/helper";
+/**
```

Comment:
This file was added in the root of the repo in the previous PR, which is wrong, moved to the right dir.

Reply:
I think we just duplicated each other's work 😊: https://github.com/gitpod-io/gitpod/pull/11458

Reply:
I've landed yours, will rebase this one.